filters/colorization/ColorizationFilter.cpp

```diff
@@ -183,72 +161,25 @@ void ColorizationFilter::ready(PointTableRef table)
 
 void ColorizationFilter::filter(PointView& view)
 {
-    int32_t pixel(0);
-    int32_t line(0);
-
-    std::array<double, 2> pix = { {0.0, 0.0} };
+    std::vector<double> data;
+    int i(0);
     for (PointId idx = 0; idx < view.size(); ++idx)
     {
+        int i(0);
         double x = view.getFieldAs<double>(Dimension::Id::X, idx);
         double y = view.getFieldAs<double>(Dimension::Id::Y, idx);
 
-        if (!getPixelAndLinePosition(x, y, m_inverse_transform, pixel,
-                line, m_ds))
-            continue;
+        bool bRead = m_raster->read(x, y, data);
+
+        if (!bRead) continue;
```

Comment:
I think we're generally avoiding one-line ifs and I think we've also pretty much gotten away from the bBoolean bit.  Is there some good reason to do it?